README.md

```diff
@@ -77,9 +77,9 @@ and using a configuration file (`bumpversion.cfg`) for more complex multi-file o
 
   The file that will be modified.
 
-  If not given, the list of `[bumpversion:file:…]` sections from the
-  configuration file will be used. If no files are mentioned on the
-  configuration file either, then no files will be modified.
+  This file is added to the list of files specifid in `[bumpversion:file:…]`
+  sections from the configuration file. If you want to rewrite only this file
```

Comment:
I think `this file` becomes ambiguous since there's now 2 files: the passed in file and the configuration file.
```suggestion
  sections from the configuration file. If you want to rewrite only files specified on the command line,
```